packages/components/src/SubHeaderBar/InputTitleSubHeader/TitleSubHeader.component.js

```diff
@@ -52,13 +56,15 @@ function TitleSubHeader({ title, subTitle, onEdit, t }) {
 
 TitleSubHeader.propTypes = {
 	title: PropTypes.string.isRequired,
-	onEdit: PropTypes.func.isRequired,
+	editable: PropTypes.bool.isRequired,
```

Comment:
`.isRequired` this prevent the use of props without assigning a boolean value

see https://reactjs.org/docs/jsx-in-depth.html#props-default-to-true